app/src/main/java/org/kin/kineticdemo/MainActivity.kt

```diff
@@ -42,9 +42,10 @@ class MainActivity : AppCompatActivity() {
 
         kineticNetworkScope.launch {
             kinetic = KineticSdk.setup(
-                "http://staging.kinetic.host",
-                "devnet",
-                1
+                endpoint = "https://sandbox.kinetic.host",
+                environment = "devnet",
+                index = 1,
+                headers = mapOf("kinetic-custom-header" to "Yay nice!"),
```

Comment:
Was it intentional adding the `paramName = ` in places or is that copilot? Generally we leave them out on Android as the IDE shows them for you.

Reply:
Copilot, of course... 😇 I removed them.